tests/plugins/autocomplete/autocomplete.js

```diff
@@ -232,6 +232,8 @@
 			editable.fire( 'keydown', new CKEDITOR.dom.event( { keyCode: 9 } ) ); // TAB
 			assert.areEqual( '', editor.getData(), 'Tab caused insertion' );
 
+			// Model may be deactivated by the previous keydown.
+			ac.model.isActive = true;
```

Comment:
If the issue is caused by `tableselection` plugin, then it'll be better to just remove this plugin from test via `removePlugins` config option. That way we'll keep the test clean.